trio_websocket/_impl.py

```diff
@@ -38,7 +38,7 @@
 CONN_TIMEOUT = 60 # default connect & disconnect timeout, in seconds
 MESSAGE_QUEUE_SIZE = 1
 MAX_MESSAGE_SIZE = 2 ** 20 # 1 MiB
-RECEIVE_BYTES = 4 * 2 ** 10 # 4 KiB
+DEFAULT_RECEIVE_BYTES = 4 * 2 ** 10 # 4 KiB
```

Comment:
the others are also defaults but not named this way, so I'd rather leave the name as is